test/test_p4est_2d.jl

```diff
@@ -587,9 +587,9 @@ end
     u = Trixi.wrap_array(u_ode, semi)
     du = Trixi.wrap_array(du_ode, semi)
     drag = Trixi.analyze(drag_coefficient, du, u, tspan[2], mesh, equations, solver,
-                         semi.cache)
+                         semi.cache, semi)
     lift = Trixi.analyze(lift_coefficient, du, u, tspan[2], mesh, equations, solver,
```

Comment:
I think that the final time needs to be increased to ensure that AMR has an effect in this test.

Reply:
True, I also recognized this when updating the test.

Reply:
I recommend that we use for the test a different AMR callback which is more sensitive and equip it with a much lower interval to keep the runtime of the test low.

Reply:
By reducing the AMR interval the AMR callback gets now called twice.